### PR TITLE
Board page redesign

### PR DESCRIPTION
**Pull request by HelloWorld017**

### CODE REVIEW

File: src/components/LikeButton.vue [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="like-button" :class="{ 'like-button--enabled': voteEnabled }">
```
Comment (litmify) [inline src/components/LikeButton.vue:2]: I think _isVoteEnabled_ is easier to understand.

File: src/bulma/index.scss [hunk 2]
```diff
@@ -7,10 +7,6 @@ $primary: $theme-red;
 $link: $grey-dark;
 $link-hover: $black-bis;
 
-a:hover {
-  text-decoration: underline;
```
Comment (litmify) [inline src/bulma/index.scss:11]: Is this rule applying for all hovered anchors?

File: src/components/LikeButton.vue [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      },
+
+      voteFn: {
+        type: Function
+      }
+    },
+
+    computed: {
+      liked() { return this.item.my_vote === true },
+      disliked() { return this.item.my_vote === false },
```
Comment (litmify) [inline src/components/LikeButton.vue:33]: Please comment type of `this.item.my_vote` .

File: src/bulma/index.scss (hunk 2 above)
Comment (HelloWorld017) [reply src/bulma/index.scss:11]: Actually, yes but as our new design doesn't require underline on hover no more, so I have removed.

**Pull request merged**